docs/source/configuration.rst

```diff
@@ -57,24 +57,47 @@ Your server/docker container clock should also match the accurate time of day!
     navigation bar! All other timestamps are in the same time zone!
 
 
-Using Amazon SES instead of SMTP email
---------------------------------------
-
+E-mail settings
+---------------
 Kiwi TCMS supports email notifications which by default are sent over SMTP and
-need to be configured via the following settings::
+need to be configured via the following settings:
 
-    # standard Django settings
-    EMAIL_HOST = 'smtp.example.com'
-    EMAIL_PORT = 25
-    DEFAULT_FROM_EMAIL = 'kiwi@example.com'
+- Common settings::
 
+    DEFAULT_FROM_EMAIL = 'kiwi@example.com'
     # additional Kiwi TCMS setting
     EMAIL_SUBJECT_PREFIX = '[Kiwi-TCMS] '
 
+- SMTP specific settings::
+
+    EMAIL_HOST = 'smtp.example.com'
+    EMAIL_PORT = 25
+    EMAIL_HOST_USER = 'smtp_username'
+    EMAIL_HOST_PASSWORD = 'smtp_password'
+
+To enable SSL or TLS support include one of the following::
+
+    EMAIL_USE_TLS = True
+    EMAIL_USE_SSL = True
+
+For more details refer to Django documentation at https://docs.djangoproject.com/en/3.0/topics/email/
+
+
+Testing and debugging e-mail configuration
+------------------------------------------
+Sending test e-mail to one or more addresses::
+
+    ./manage.py sendtestemail user1@example1.tld user2@example2.tld ...
```

Comment:
Use `docker exec -it kiwi_web /Kiwi/manage.py ....` for consistency with the rest of the docs.